tox.ini

```diff
@@ -8,7 +8,7 @@ deps =
     pytest
 commands =
     pylint --rcfile .pylintrc locust_plugins/
-    pylint --rcfile .pylintrc examples/
+    python -c 'from glob import glob; import subprocess; subprocess.check_call(["pylint", "--rcfile", ".pylintrc"] + glob("examples/*.py"))'
```

Comment:
Tox doesn't support globing of files
https://github.com/tox-dev/tox/issues/1571